sdk/core/Azure.Core.Experimental/tests/DynamicJsonEnumerableTests.cs

```diff
@@ -84,5 +84,56 @@ public void CanForEachOverIntArray()
                 Assert.AreEqual(expected++, i);
             }
         }
+
+        [Test]
```

Comment:
Might be nice to also add a test where the object to enumerate is nested in another property.

Reply:
Sounds good - will do!